src/app/locale/en_US/Alma_Installments.csv

```diff
@@ -54,4 +54,13 @@
 "Installments count","Installments count"
 "Paid amount (%1) does not match due amount (%2) for order %3","Paid amount (%1) does not match due amount (%2) for order %3"
 "Payment state incorrect (%s & %s) for order %s","Payment state incorrect (%s & %s) for order %s"
-"Order canceled by customer","Order canceled by customer"
\ No newline at end of file
+"Order canceled by customer","Order canceled by customer"
```

Comment:
did you already updated crowdin with this new source keys ?

Reply:
No i am waiting for Romain QA